corehq/apps/enterprise/tests/test_forms.py

```diff
@@ -111,7 +111,7 @@ def test_update_settings_clears_cache(self, clear_domain_caches):
         domains_cache_cleared = [
             arg[0][0] for arg in clear_domain_caches.call_args_list
         ]
-        self.assertListEqual(
+        self.assertCountEqual(
```

Comment:
I agree, `assertCountEqual` is a confusing name. Could also do the same with a set comparison.
```py
        domains_cache_cleared = {
            arg[0][0] for arg in clear_domain_caches.call_args_list
        }
        self.assertEqual(
            domains_cache_cleared,
            {'test-emws-form-001', 'test-emws-form-002'},
        )
```

Reply:
Yeah that is how I've been addressing these, but felt like there must be some method in existence that handles lists directly. Despite the confusing name, I think I still prefer using it as it is very explicit about order not mattering _and_ catches unexpected duplicates if that is applicable.

Reply:
I think it's a lot less efficient than set comparison, but in this case that does not matter.